build/parseSpec.c

```diff
@@ -883,30 +908,63 @@ static int parseEmpty(rpmSpec spec, int prevParsePart)
     return res;
 }
 
+static int parsePostbuild(rpmSpec spec)
+{
+    int res = PART_ERROR;
+    int rc;
+    char *line;
+
+    if (spec->postbuild != NULL) {
+	rpmlog(RPMLOG_ERR, _("line %d: second %%postbuild\n"),
+	       spec->lineNum);
+	goto exit;
+    }
+
+    spec->postbuild = newStringBuf();
+
+    line = spec->line + sizeof("%postbuild") - 1;
+    SKIPSPACE(line);
+    if (line[0] != '\0') {
+	rpmlog(RPMLOG_ERR,
+	    _("line %d: %%postbuild doesn't take any arguments: %s\n"),
+	    spec->lineNum, spec->line);
+	goto exit;
+    }
+
+    while (1) {
+	if ((rc = _readLine(spec, 0, 1)) > 0) {
+	    res = PART_NONE;
+	    break;
+	} else if (rc < 0) {
+	    res = PART_ERROR;
+	    break;
+	}
+	if (!strncmp(spec->line, "%%end",  5)) {
+	    spec->line++;
+	    res = PART_EMPTY;
+	    break;
+	}
+	appendStringBufAux(spec->postbuild, spec->line, 0);
+    }
+    printf("XXX\n%s\nXXX\n", getStringBuf(spec->postbuild));
```

Comment:
at least I know that you are using good old printf-everywhere when writing new code :D

Reply:
You say that as if there was some other way :open_mouth: